### ISSUE TITLE
[qt-journalist-updater] Run securedrop-admin setup

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#3284]
## Description

We rarely _need_ `securedrop-admin update` to be run (though we do in 0.7 as we're adding a dependency to `securedrop-admin`), but since it doesn't hurt, let's run it for each update. We can do this by adding a `QThread` to run `securedrop-admin setup`. Also update the time estimates provided in the updater. 

## User Stories

As a SecureDrop user, I don't want to have to manually run `./securedrop-admin setup` on the command line if everything else in the update happens automatically

**Comment by kushaldas** [comment]
I think @redshiftzero [meant](https://github.com/freedomofpress/securedrop/pull/3283/files#r182489560) `./securedrop-admin setup` not update.

**Comment by conorsch** [comment]
Yes, I agree @kushaldas! `./securedrop-admin setup` is what should be run as part of `./securedrop-admin update`.

**Comment by kushaldas** [comment]
The `./securedrop-admin setup` is taking around 3-4 minutes in my vms. I am adding 4 minutes extra in the time estimate as the first step, before the tool actually updates.

**Comment by conorsch** [comment]
> The `./securedrop-admin setup` is taking around 3-4 minutes in my vms. I

But only on _first_ run, right? If no changes are required, then it only takes a few seconds for me.

**Comment by kushaldas** [comment]
>But only on first run, right? If no changes are required, then it only takes a few seconds for me.
Yes, otherwise it is taking around 50 seconds in my vm.

**Comment by kushaldas** [comment]
@redshiftzero I have pushed the code in `qt-run-setup-first` branch. But, I am feeling that I am missing something. Please make any changes required and create the PR.

**Comment by redshiftzero** [comment]
Ah, woops, sorry, yes, I meant `securedrop-admin setup` :)

**Comment by redshiftzero** [comment]
Regarding the time elapsed during `setup`, I tested a few times in a Tails VM, and I agree with y'all: the time `securedrop-admin setup` takes seems to be <1 min. I also tested the time taken to _update_ the dependencies (for a version like 0.7 where we add a dependency) and that a few seconds longer but still approximately a minute ✅

One thing I noticed - we want to run `securedrop-admin setup` _after_ the update. The reason is that we can safely assume `securedrop-admin setup` has been run on the current version. If `securedrop-admin setup` has not been run, the user will not have updated to the version of SecureDrop where the Updater GUI automatically appears when updates are needed. This is because updating requires the execution of `securedrop-admin tailsconfig`.
